### PR TITLE
Autofill address when creating space

### PR DESCRIPTION
**Pull request by mathias234**
No design work has been done here, it still uses the  green create space CRUD on the main page

It works by the frontend sending a query to the backend which uses Geonorge API to figure out the lat lng of the typed address. if there is no javascript enabled on the frontend the fields will not automatically fill out but the backend will still try to figure out the coordinates based on what was submitted in the form.

### CODE REVIEW

**Review by DanielJackson-Oslo — APPROVED** [review]
Looks very good to me! I've created a PR with some suggested changes I made while reviewing: https://github.com/lnu-norge/lokaler.lnu.no/pull/61

Would love for those to be reviewed in as well, though they can be a seperate PR if you'd like to.

File: app/controllers/spaces_controller.rb [hunk 1]
```diff
@@ -11,12 +11,29 @@ def show
   end
 
   def create
-    space_type = SpaceType.find_or_create_by!(type_name: 'Skole')
-    @space = Space.create!(space_type: space_type, **space_params)
+    address_params = get_address_params(params)
+    return redirect_to spaces_path, alert: 'Unable to find address' if address_params.nil?
```
Comment (asolvang) [inline app/controllers/spaces_controller.rb:15]: Kanskje legge alert teksten til oversettelses filen?

**Review by asolvang — APPROVED** [review]
LGTM

File: app/controllers/spaces_controller.rb (hunk 1 above)
Comment (DanielJackson-Oslo) [reply app/controllers/spaces_controller.rb:15]: Bra sett! Burde definitivt være der.

**Pull request merged**